desktop/ui/src/main/java/org/datacleaner/widgets/visualization/JobGraph.java

```diff
@@ -278,9 +284,31 @@ public void paint(Graphics g) {
                             + "So add one or more such components.";
                     imagePath = "images/window/canvas-bg-plus.png";
                 } else {
-                    title = null;
+                    title=null;
                     subTitle = null;
                     imagePath = null;
+                    
+                    try {
+                        if (!_analysisJobBuilder.isConfigured(true)) {
+                            subTitle="Job is not correctly configured";
+                            warning=true; 
+                        }
+                    } catch (Exception ex) {
+                        logger.debug("Job not correctly configured", ex);
+                        final String errorMessage;
+                        if (ex instanceof UnconfiguredConfiguredPropertyException) {
+                            UnconfiguredConfiguredPropertyException unconfiguredConfiguredPropertyException = (UnconfiguredConfiguredPropertyException) ex;
+                            ConfiguredPropertyDescriptor configuredProperty = unconfiguredConfiguredPropertyException
+                                    .getConfiguredProperty();
+                            ComponentBuilder componentBuilder = unconfiguredConfiguredPropertyException.getComponentBuilder();
+                            errorMessage = "Property '" + configuredProperty.getName() + "' in "
+                                    + LabelUtils.getLabel(componentBuilder) + " is not set!";
```

Comment:
I like it, but would like to refine it a bit more :-)

How about "Please set X on Y to continue".

My point being that "property" and "component" are quite technical words and might not resonate with the user. The user will just see a graph node called "Y" and when he double clicks he will see a label for "X". The fact that we call these components and properties are "nice to know" but if we can make meaningful messages to the user without using those words, then I think it's an advantage.